frontend/src/app/main/data/workspace/libraries_helpers.cljs

```diff
@@ -1354,9 +1379,51 @@
                      (conj roperations roperation)
                      (conj uoperations uoperation)))))))))
 
+(defn- propagate-attrs
+  "Helper that puts the src-shape attributes (attrs) into tgt-shape but only if
+  not touched the group or if omit-touched? flag is true"
+  [tgt-shape src-shape attrs omit-touched?]
+  (let [touched (get tgt-shape :touched #{})]
+    (->> attrs
+         (reduce
+          (fn [tgt-shape attr]
+            (let [attr-group (get ctk/sync-attrs attr)]
+              (cond-> tgt-shape
+                (or (not (touched attr-group)) (not omit-touched?))
+                (assoc attr (get src-shape attr)))))
+          tgt-shape))))
+
+(defn- update-flex-child-copy-attrs
+  "Synchronizes the attributes inside the flex-child items (main->copy)"
+  [changes shape-main shape-copy main-container main-component copy-container omit-touched?]
+  (-> changes
+      (pcb/with-container copy-container)
+      (pcb/with-objects (:objects copy-container))
```

Comment:
The container and objects should already have been added to the changes by this moment. I think it's dangerous setting them here, for it can affect other changes later in the same chain.

Reply:
All the current code is not using the changes builder that's why it doesn't have the objects/container (that only will set up the metadata).

Reply:
fix